src/s1reader/s1_reader.py

```diff
@@ -486,7 +486,7 @@ def load_bursts(path: str, orbit_path: str, swath_num: int, pol: str='vv',
             warnings.warn("None of provided bursts IDs found")
```

Comment:
I your example what do you get if you loop over multiple sub-swaths:
```
for i_subswath in [1,2]:
     s1reader.load_bursts(path, orbit_path, i_subswath, pol, 9)
```
I see that the warning once while I think we would have expected to see it twice. Any idea what is going on? Replacing warning with a simple print message leads to the correct behavior tgough!